### PR TITLE
Printing SQS Write + S3 File Upload

### PR DESCRIPTION
**Pull request by allen-tran**
This pull request brings the Printer.js that uploads files to S3, grabs the link to the S3 object, and writes the message to SQS. The VM has an SQS read that will grab this link to download the file locally.

### CODE REVIEW

**Comment by andrewtedi** [comment]
mem

File: api/logging_api/routes/Printer.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+AWS.Credentials(s3BucketKeys.AWSACCESSKEYID, s3BucketKeys.AWSSECRETKEY);
+
+AWS.config.update({
+  region: 'us-west-1',
+  endpoint: 'https://s3.amazonaws.com',
+  credentials: creds
+});
+
+router.get('/healthCheck', (req, res) => {
+  // console.log(`Hello ${req.query.name}!`);
```
Comment (evanugarte) [inline api/logging_api/routes/Printer.js:23]: remove this and the comments below

File: api/logging_api/routes/Printer.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+  res.sendStatus(OK);
+});
+app.use(express.json());
```
Comment (evanugarte) [inline api/logging_api/routes/Printer.js:26]: i don't think you need this. if the requests still work the same without it remove

File: api/logging_api/server.js [hunk 3]
```diff
@@ -4,9 +4,11 @@ function main() {
   const API_ENDPOINTS = [
     __dirname + '/routes/ErrorLog.js',
     __dirname + '/routes/PrintLog.js',
-    __dirname + '/routes/SignLog.js'
+    __dirname + '/routes/SignLog.js',
+    __dirname + '/routes/Printer.js'
+
   ];
-  const loggingServer = new SceHttpServer(API_ENDPOINTS, 8081, '/logapi/');
+  const loggingServer = new SceHttpServer(API_ENDPOINTS, 8081, '/meme/');
```
Comment (evanugarte) [inline api/logging_api/server.js:11]:
```suggestion
  const loggingServer = new SceHttpServer(API_ENDPOINTS, 8081, '/logging_api/');
```

File: src/APIFunctions/2DPrinting.js [hunk 4]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable */
```
Comment (evanugarte) [inline src/APIFunctions/2DPrinting.js:1]: why

File: src/APIFunctions/2DPrinting.js [hunk 5]
```diff
@@ -12,6 +13,11 @@ let LOGGING_API_URL = process.env.NODE_ENV === 'production' ?
 let RPC_API_URL = process.env.NODE_ENV === 'production' ?
   config.RPC_API_URL_PROD : config.RPC_API_URL;
 
+  console.log({
```
Comment (evanugarte) [inline src/APIFunctions/2DPrinting.js:16]: i thought we said no console logs :,(

**Review by evanugarte — APPROVED** [review]
address comments, looks good

File: src/APIFunctions/2DPrinting.js (hunk 5 above)
Comment (allen-tran) [reply src/APIFunctions/2DPrinting.js]: Addressed.

File: src/APIFunctions/2DPrinting.js (hunk 4 above)
Comment (allen-tran) [reply src/APIFunctions/2DPrinting.js]: Addressed.

File: api/logging_api/routes/Printer.js (hunk 2 above)
Comment (allen-tran) [reply api/logging_api/routes/Printer.js]: Works.

**Review by ajha7 — APPROVED** [review]

**Review by andrewtedi — APPROVED** [review]

File: api/logging_api/routes/Printer.js [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+AWS.Credentials(s3BucketKeys.AWSACCESSKEYID, s3BucketKeys.AWSSECRETKEY);
+
+AWS.config.update({
+  region: 'us-west-1',
+  endpoint: 'https://s3.amazonaws.com',
+  credentials: creds
+});
+
+router.get('/healthCheck', (req, res) => {
+  res.sendStatus(OK);
```
Comment (evanugarte) [inline api/logging_api/routes/Printer.js:21]: this will be improved later

**Pull request merged**